Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-explain.txt

MongoDB\\Collection::explain()
==============================

.. meta::

>    :description: Explore how to use `MongoDB\Collection::explain()` to analyze command execution details, including parameters and return values.

Reviewer: ```suggestion
   :description: Analyze command execution details with the MongoDB PHP Library, including parameters and return values.
```
